tests/ALCProxy.TestInterface/IExternalClass.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace ALCProxy.TestInterface
```

Comment:
Maybe this is acceptable, why do we have small test projects with single types in them? TestInterface, TestInterfaceUpdated? Could they be in one instead?

Reply:
The test here is to check if a plugin that was built off an older interface (`TestInterface`) can be proxied with a newer version of the interface and treated like one (`TestInterfaceUpdated`). Keeping them in separate assemblies is to simulate the main ALC where the proxy is created not having any access to the older interface, but I could probably merge them into one assembly to organize it a bit better.

Reply:
Actually whoops, my mistake. Since they're technically the same type I'm not sure we could merge them into one assembly.